app/src/views/ResultPage.vue

```diff
@@ -406,6 +407,9 @@
   @import "~leaflet.markercluster/dist/MarkerCluster.Default.css";
   @import "~leaflet-gesture-handling/dist/leaflet-gesture-handling.css";
 
+   .hide-copyright strong{
```

Comment:
This will result in all strong tags (also from all pages we crawl in the future) to be invisible.
Please only hide the shameOnYouIfYouRemove class.